src/main/resources/scripts/announcements/generate_json.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/python
+import argparse
+import json
+import mysql.connector
+import re
+
+# This script will fetch all the rows from the announcement table, then parse
+# the title out from the message content and return a json file with the newly
+# populated title and modified message content.
+#
+# Step #1: Run this script to generate a json file with the modified
+# announcement data.
+#
+# Step #2: Please review the json file to make sure the title was parsed
+# correctly from the message content.
+#
+# Step #3: To update the announcement table with the json file run the
+# update_announcement.py script next.
```

Comment:
Please add a note in UPGRADING.md about this script.  You should consider adding something to the Upgrade Notes docs page about it if the notice in UPGRADING.md gets too long.  https://phac-nml.github.io/irida-documentation/administrator/upgrades/

Reply:
I hope this works. Please see cd235201d1c5181778d4c564fe81992004722137.